### ISSUE TITLE
I can't see any Quotes...

### ISSUE THREAD

**Issue by MuhammadShahzeb123** [joelshepherd/tabliss#652]
I have been using tablets for a while now but I cannot use the quotes. I don't know, but whenever I add that to my desktop, it doesn't show up.

Please help

**Comment by cpiprint** [comment]
So true...bug or fix?

**Comment by tinkermaniac** [comment]
me tooo, only the developer excuses works...

**Comment by cod378** [comment]
That is because "They Said So" require a token authentication, but the extension haven't a way to handle this. We must create/register a free account in theysaidso.com to get our personal token (its free)... and wait for the extension devs implements a way to put the token.

**Comment by MatrixRanger98** [comment]
yeah could this be fixed? any workround is also good. I like the quotes

**Comment by marcoangelo** [comment]
Any updates for this? Shame as Id quite like this feature to work

**Comment by shokk** [comment]
Switching to quotes.net might be easier and everyone can get their own key
